packages/features/bookings/Booker/Booker.tsx

```diff
@@ -128,11 +128,8 @@ const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: Book
         <div
           ref={animationScope}
           className={classNames(
-            // Size settings are abstracted on their own lines purely for readbility.
-            "[--booker-main-width:480px] [--booker-timeslots-width:240px] lg:[--booker-timeslots-width:280px]",
-            layout === "small_calendar" && "[--booker-meta-width:240px] lg:[--booker-meta-width:280px]",
-            layout !== "small_calendar" && "[--booker-meta-width:340px] lg:[--booker-meta-width:424px]",
-            // Other styles
+            // Sets booker size css variables for the size of all the columns.
+            ...getBookerSizeClassNames(layout, bookerState),
```

Comment:
Decided to move this to the config file so the booker component itself becomes a bit cleaner. 
Also that way we can use some more comments in the config file, since they won't bother anyone there. See next file.